docs/UserGuide.md

```diff
@@ -145,15 +145,34 @@ Examples:
 
 Adds a remark to an existing patient in the patient book for nurses to store additional data unique to the patient.
 
-Format: `remark INDEX r/REMARK`
+Format: `remark INDEX [r/REMARK]`
 
-* Creates and adds a remark for the patient at the specified `INDEX`. The index refers to the index number shown in the displayed patient list. The index **must be a positive integer** 1, 2, 3, …​
-* A remark serves as an optional field for a patient and can be modified using the edit command after it has been created.
-
-Examples:
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+We implemented our Remarks feature with the hopes of empowering you with the ability to add extra information to a patient's bio data! 
+So don't be shy and feel free to add anything under the sun that you feel apt for the patient. :smile:
+</div>
+<br> 
+![result for 'Add remark'](images/addRemark.jpg)
+<br>
+* Creates and **adds a remark** for the patient at the specified `INDEX`. The index here refers to the index number shown on the left side of the displayed patient list. 
+* Please note that the index used **must be a positive integer** 1, 2, 3, …​
+* A remark serves as an **optional field** for adding extra info to a patient's bio data and can be left blank if it is not applicable.  modified using the edit/remark command after it has been created.
```

Comment:
I think the second line can be omitted. (Addressed in the 5th point)

Reply:
oh, thanks for spotting this bro 👍🏻 missed it out when I was editing the code haha.

Reply:
oh, thanks for spotting this bro 👍🏻 missed it out when I was editing the code haha.